helm/superset/Chart.yaml

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 #
 apiVersion: v2
-appVersion: "2.1.0"
+appVersion: "2.1.1"
```

Comment:
@craig-rueda I see I have updated the wrong version. Glad these checks are in place.
I'll fix tomorrow

Reply:
@craig-rueda that's fixed now